addons/account_payment/wizards/payment_refund_wizard.py

```diff
@@ -58,8 +58,9 @@ def _compute_amount_to_refund(self):
     @api.depends('transaction_id.provider_id', 'transaction_id.payment_method_id')
     def _compute_support_refund(self):
         for wizard in self:
-            p_support_refund = wizard.transaction_id.provider_id.support_refund
-            pm = wizard.transaction_id.payment_method_id
+            tx_sudo = wizard.transaction_id.sudo() # needed for users without access to the provider
```

Comment:
```suggestion
            tx_sudo = wizard.transaction_id.sudo()  # needed for users without access to the provider
```
Not sure why the ci/style check isn't flagging this [despite catching it](https://runbot163.odoo.com/runbot/static/build/74412478-17-0/logs/check_style_ruff-ruff-output.json), but two spaces are needed for Python inline comments ([E261](https://docs.astral.sh/ruff/rules/too-few-spaces-before-inline-comment/))